Update EOS examples for the new api on the Jungle 2.0 testnet (#2)
oraclize/eos-examples: random-datasource/README.md

# Oraclize's Random Datasource Example 

This repo is to demonstrate how you would work with the Oraclize **random** datasource.

> The `randomsample` example allows you to *retrive a random number*.

Reviewer: ```suggestion
The `randomsample.cpp` example allows you to *retrieve a random number*.
```

## Notes

> Oraclize replies to your `oraclize_newRandomDSQuery` with the *random number result* calling your `callback(...)`.

Reviewer: ```suggestion
Oraclize replies to your `oraclize_newRandomDSQuery` by calling your `callback(...)` with the *random number result*.
```

You can search your transaction ID in one of the following links to verify it:

> * :mag_right::ledger: [Blocks.io](https://jungle.bloks.io/): a block explorer for the Jungle 2.0 testnet.

Reviewer: ```suggestion
* :mag_right::ledger: [Blocks.io](https://jungle.bloks.io/): A block explorer for the Jungle 2.0 testnet.
```

> * :palm_tree::lion::palm_tree: [Jungle 2.0](https://monitor.jungletestnet.io/#home): a transaction explorer is available selecting *Get TX* on the Jungle 2.0 testnet website.

Reviewer: ```suggestion
* :palm_tree::lion::palm_tree: [Jungle 2.0](https://monitor.jungletestnet.io/#home): A transaction explorer is available by selecting *Get TX* on the Jungle 2.0 testnet website.
```
